app/api/files/jsRoutes.js

```diff
@@ -127,6 +127,7 @@ const routes = app => {
       },
       proxyReqOptDecorator(proxyReqOpts) {
         const { tenant, ...headers } = proxyReqOpts.headers;
+        delete headers.cookie;
         return {
           ...proxyReqOpts,
           headers: { ...headers },
```

Comment:
@mercy can we instead of modifying headers, only delete it on the new returned object ?
`
headers: { ...headers, cookie: undefined or {} }
`
this way we avoid side effects.

another option is change line 129 to 
`
const { tenant, cookie, ...headers } = proxyReqOpts.headers;
`

Reply:
Sure thanks, I changed to the second option.